src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/altar/AncientPedestal.java

```diff
@@ -104,7 +119,7 @@ private boolean testItem(@Nullable Entity n) {
         }
     }
 
-    public @Nonnull ItemStack getOriginalItemStack(@Nonnull Item item) {
+    public @Nonnull ItemStack getOriginalItemStack(@Nonnull Item item, @Nonnull Block block) {
```

Comment:
Is this new added param used for anything?

Reply:
it was but is no longer, will remove

Reply:
fixed in next commit